src/HttpAsyncClientDiscovery.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Http\Discovery;
+
+/**
+ * Finds an HTTP Asynchronous Client
+ *
+ * @author Joel Wurtz <joel.wurtz@gmail.com>
+ */
+class HttpAsyncClientDiscovery extends ClassDiscovery
+{
+    /**
+     * @var HttpAsyncClient
```

Comment:
Like i said it's done, github just don't show it here :) as it's from a old commit